x-pack/plugins/security_solution/public/resolver/test_utilities/simulator/index.tsx

```diff
@@ -175,14 +175,24 @@ export class Simulator {
   }
 
   /**
-   * Return an Enzyme ReactWrapper for any child elements of a specific processNodeElement
-   *
-   * @param entityID The entity ID of the proocess node to select in
-   * @param selector The selector for the child element of the process node
+   * The button that opens a node's submenu.
    */
-  public processNodeChildElements(entityID: string, selector: string): ReactWrapper {
+  public processNodeSubmenuButton(
+    /** nodeID for the related node */ entityID: string
+  ): ReactWrapper {
     return this.domNodes(
-      `${processNodeElementSelector({ entityID })} [data-test-subj="${selector}"]`
+      `[data-test-subj="resolver:submenu:button"][data-test-resolver-node-id="${entityID}"]`
+    );
+  }
+
+  /**
+   * The primary button (used to select a node) which contains a label for the node as its content.
+   */
+  public processNodePrimaryButton(
```

Comment:
Node's have 2 buttons, one to select it, one to open its submenu. This is the one that selects it.

Reply:
Same comment as above, but if we decide to stay this route, would a name like `processNodeShowDetailsButton` work? But then again, that could be confusing considering the process cube is also clickable as well and has the same effect?

Reply:
That name could also work. We could also drop `process` from the name. We could also do something involving aria roles somehow. For example, if the graph has a test  subj, we could select `[role="treeitem"]` descendants to get the node wrapper elements. We could also select some other `role` to get this 'primary' button perhaps. I don't know enough about it yet.

Reply:
aria attributes could work. The primary reason I lean towards naming this the effect the button has is that it provides the reader an initial idea of what any related tests should be doing.

Reply:
I see what you mean. Related idea: perhaps we could have a `selectNodeViaButtonClick` method.